docs/reference/config-template.txt

```diff
@@ -99,7 +99,7 @@ Consider the experiment configuration below:
 
 You may find that the values for the ``checkpoint_storage`` field are the same
 for many experiments and you want to use a configuration template to reduce the
-redundancy. In this example, such a template can be written as follows:
+redundancy. You might write a template like the following:
```

Comment:
question: Do we do a lot of second-person in the documentation?

Reply:
Yep, I'm seeing 300+ instances currently.